tests/backends/test_ddp.py

```diff
@@ -68,3 +72,36 @@ def test_multi_gpu_model_ddp_fit_test(tmpdir, cli_args):
     model_outs = result['result']
     for out in model_outs:
         assert out['test_acc'] > 0.90
+
+
+# START: test_cli ddp test
+@pytest.mark.skipif(os.getenv("PL_IN_LAUNCHER", '0') == '1', reason="test runs only in DDPLauncher")
```

Comment:
can we rather have a function in init which would wrap this?
```
def _run_ddp():
```suggestion
@pytest.mark.skipif(not _run_ddp(), reason="test runs only in DDPLauncher")
```
  return os.getenv("PL_IN_LAUNCHER", '0') == '1'
```